django_app/redbox_app/redbox_core/consumers.py

```diff
@@ -92,6 +92,9 @@ class ChatConsumer(AsyncWebsocketConsumer):
     redbox = Redbox(env=get_settings(), debug=True)
     chat_message = None  # incrementally updating the chat stream
 
+    # flake8: noqa: C901
+    # flake8: noqa: PLR0915
```

Comment:
Can we have a backlog ticket to resolve the skipped checks C901 and PLR0915 please? Its unfair to make a refactor of this method part of this unit test ticket - and should be treated as a work item in its own right.